ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh

```diff
@@ -27,21 +27,24 @@ SSHOPTS="-o ConnectTimeout=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/
 
 function getlogs() {
   echo "### Downloading logs..."
-  ssh $SSHOPTS root@$IP tar -czC "/tmp/artifacts/must-gather" -f "/tmp/artifacts/must-gather.tar.gz" .
-  scp $SSHOPTS root@$IP:/tmp/artifacts/must-gather.tar.gz ${ARTIFACT_DIR}
-  scp $SSHOPTS root@$IP:/var/tmp/sosreport-\* ${ARTIFACT_DIR}
+  scp $SSHOPTS root@$IP:/tmp/artifacts/\*.tar\* ${ARTIFACT_DIR}
 }
 
 # Gather logs regardless of what happens after this
 trap getlogs EXIT
 
 echo "### Gathering logs..."
 timeout -s 9 15m ssh $SSHOPTS root@$IP bash - << EOF |& sed -e 's/.*auths.*/*** PULL_SECRET ***/g'
+cd dev-scripts
+
+# Get install-gather, if there is one
+cp log-bundle\* /tmp/artifacts || true
```

Comment:
It's only available on bootstrap failure, so it's fine if it doesn't exist.  It'll be [a file named log-bundle-XXXX.tar.gz if it does](https://github.com/openshift/installer/blob/master/data/data/bootstrap/files/usr/local/bin/installer-gather.sh#L133).